src/data/results-display.js

```diff
@@ -106,7 +106,9 @@
     if (type.startsWith("0.5") || type.startsWith("1/2") || type.startsWith("0,5")) {
       nbHours = 4;
     }
-    type = /[A-Z]+/.exec(type.toUpperCase())[0];
+    if (type) {
+      type = /[A-Z]+/.exec(type.toUpperCase())[0];
+    }
```

Comment:
If the JS error you get is that "type" is undefined, then you'll likely have a problem in the previous lines as well.